Directory.Build.props

```diff
@@ -11,5 +11,20 @@
     <SignAssembly>True</SignAssembly>
     <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)/SQL2003.snk</AssemblyOriginatorKeyFile>
     <IsPackable>false</IsPackable>
+    <Company>Microsoft</Company>
+    <Authors>Microsoft</Authors>
+    <GenerateAssemblyInfo>true</GenerateAssemblyInfo>
+    <IsPackable>true</IsPackable>
```

Comment:
This should be false in this file - we want projects to specifically opt-in to being packable in their own csproj

(we may want to look into only adding the nuget related properties when IsPackable is true, but I don't think there should be any harm in including them even if the project isn't being packaged so leaving them like this is probably fine)